Testing/xASL_test_IntegrationTesting.m

```diff
@@ -130,10 +130,8 @@
     xASL_delete(fullfile(testingRoot,'FlavorDatabaseTest',flavorName,'rawdata'),1);
     
     % Create default dataPar.json
-    defaultDataPar.x.subject_regexp = '^sub-.*$';
     defaultDataPar.x.bUseMNIasDummyStructural = 1; % when no structural data, use ASL-MNI registration
```

Comment:
I thought this wasn't the case for all flavors, you mind checking if all flavors don't have an T1.nii, otherwise we should remove any structural NIfTIs (perhaps in another issue)

Reply:
I think this isn't the case for all flavors. Jan is the expert here though. I just wanted to create an initial testing script where you can run each flavor from start to finish with less set up and stuff like in Jans version, but his one probably works better.

Reply:
Good point. This is not really needed. Some datasets have T1. And there were 1-2 datasets where the bUseMNIasDummy option was needed. But I have added the DataPar to the flavors with this option on. So for the purpose of testing flavors - we don't need it.

And we should probably exclude it so that it doesn't use MNI for those data that have T1.

We definitively do **not** want to remove T1.niis from flavors, because we want to test the import of them as well!!!

Reply:
> We definitively do not want to remove T1.niis from flavors, because we want to test the import of them as well!!!

Agree, that would make the test database worse.

> And we should probably exclude it so that it doesn't use MNI for those data that have T1.

Oh, sorry, didn't knew that you added a dataPar.json for the ones without T1.


f668525 👍

Reply:
> Good point. This is not really needed. Some datasets have T1. And there were 1-2 datasets where the bUseMNIasDummy option was needed. But I have added the DataPar to the flavors with this option on.

Are you sure about that? For example the first one (**GE_PCASL_2DEPI_volunteer**) does not have a T1 scan **and** there is no `dataPar.json` with the `x.bUseMNIasDummyStructural = 1;` setting.

Reply:
@MichaelStritt very good point :D Sorry for the confusion. I just did remember that setting this parameter wasn't necessary and that everything was running without it. But that's because we already have it in the xASL_test_Flavors.m :) I didn't realize that we need it in that other function because it is another unrelated function :D And that other parameter I was talking about is something else.

So indeed, we need to have this :) Easy solution - I will just remove your last commit and then we can merge everything. I will do that to avoid conflicts :)